csp-mixin/panels.libsonnet

```diff
@@ -899,5 +899,667 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+        },
+      ]),
+    gce_top5_disk_read_bytes:
+      this.signals.gcpceOverview.top5DiskRead.asTable(format='table')
+      + g.panel.table.standardOptions.withUnit('bytes')
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Time',
+          },
+          properties: [
+            {
+              id: 'custom.hidden',
+              value: true,
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'instance_name',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Instance',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'job',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Job',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'project_id',
+          },
+          properties: [
+            {
+              id: 'displayName',
+              value: 'Project ID',
+            },
+          ],
+        },
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value',
+          },
+          properties: [
+            {
+              id: 'custom.width',
+              value: 100,
+            },
+          ],
+        },
+      ]),
+
+    gce_instances:
+      g.panel.table.new('Instances')
+      + g.panel.table.queryOptions.withTargets(
+        [
+          this.signals.gcpceOverview.tableCpuUtilization.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableUptime.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableSentPackets.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableReceivedPackets.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableSentBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableReceivedBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableReadBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
+          this.signals.gcpceOverview.tableWriteBytes.asTarget()
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withFormat('table'),
```

Comment:
Can you show first **Disk read and write** and then **Network sent and received** columns? It’s just to have same order in GCP and Azure. And also to have the same order you use on the breakdown panels below.

Reply:
I'll keep this order because it's the same as in the below panels